code/obj/item/tank.dm

```diff
@@ -444,6 +444,31 @@ Contains:
 
 ////////////////////////////////////////////////////////////
 
+/obj/item/tank/emergency_plasma
+	name = "emergency toxintank"
+	icon_state = "em_plsma"
+	flags = FPRINT | TABLEPASS | ONBELT | CONDUCT
+	health = 5
+	w_class = W_CLASS_SMALL
+	force = 3.0
+	stamina_damage = 30
+	stamina_cost = 16
+	desc = "A small tank that is labelled to contain plasma. In emergencies, wear a mask that can be used to transfer plasma. These are only issued for those with plasmatoidism so don't loose it!"
+	wear_image_icon = 'icons/mob/clothing/belt.dmi'
+	distribute_pressure = 17 // setting these things to start at the minimum pressure needed to breathe - Haine
```

Comment:
Is this still correct? Had vague recollection of it possibly needing less but I'm unsure since the % are deferent.

Reply:
atomized PR's, i was gonna ask Adhara and Emily if this was still needed but thats not really the place of this pr

Reply:
What do you mean? I'm not asking any other instances where this might be copy-pasta, just this one.

Reply:
What do you mean? I'm not asking any other instances where this might be copy-pasta, just this one.